### PR TITLE
improve docs

### PR DESCRIPTION
**Pull request by gpedro**

### CODE REVIEW

**Comment by gpedro** [comment]
ping @AndreasHeiberg

File: README.md [hunk 1]
```diff
@@ -13,31 +13,34 @@ Currently let's you easily flash notifications to the session. It also supports
...
 ```
 
 Register the service provider in `app/config/app.php`.
 
 ```php
+# Laravel 4x
 'Andheiberg\Notify\NotifyServiceProvider',
+
+# Laravel 5x
+Andheiberg\Notify\NotifyServiceProvider::class,
```
Comment (andheiberg) [inline README.md:29]: This is not a laravel 5 feature, but granted this came in fashion by Laravel 5. Could I ask you to remove the "Laravel 4x" (the 4 lines above)?

File: README.md [hunk 2]
```diff
@@ -13,31 +13,34 @@ Currently let's you easily flash notifications to the session. It also supports
...
 ```
 
 Add the alias to the list of aliases in `app/config/app.php`.
 
 ```php
+# Laravel 4x
 'Notify' => 'Andheiberg\Notify\Facades\Notify',
+
+# Laravel 5x
+'Notify' => Andheiberg\Notify\Facades\Notify::class,
```
Comment (andheiberg) [inline README.md:35]: This is not a laravel 5 feature, but granted this came in fashion by Laravel 5. Could I ask you to remove the "Laravel 4x" (the 4 lines above)?

**Comment by andheiberg** [comment]
So sorry, I meant to merge it in and then make above changes my self. Would you mind doing it, I will be sure to merge it in :) sorry for the wait.

**Comment by andheiberg** [comment]
Thanks man!

**Comment by gpedro** [comment]
@AndreasHeiberg done.

**Comment by gpedro** [comment]
<3

**Pull request merged**